README.md

```diff
@@ -30,27 +51,249 @@ npm install --save ipfsd-ctl
 
 IPFS daemons are already easy to start and stop, but this module is here to do it from JavaScript itself.
 
+### Local node
+
 ```js
 // Start a disposable node, and get access to the api
-// print the node id, and kill the temporary daemon
-
-// IPFS_PATH will point to /tmp/ipfs_***** and will be
-// cleaned up when the process exits.
+// print the node id, and stop the temporary daemon
 
-var ipfsd = require('ipfsd-ctl')
+const DaemonFactory = require('ipfsd-ctl')
+const df = DaemonFactory.create()
 
-ipfsd.disposableApi(function (err, ipfs) {
-  ipfs.id(function (err, id) {
+df.spawn(function (err, ipfsd) {
+  ipfsd.api.id(function (err, id) {
     console.log(id)
-    process.exit()
+    ipfsd.stop()
   })
 })
 ```
 
-If you need want to use an existing ipfs installation you can set `$IPFS_EXEC=/path/to/ipfs` to ensure it uses that.
+### Remote node
+
+```js
+// Start a remote disposable node, and get access to the api
+// print the node id, and stop the temporary daemon
+
+const DaemonFactory = require('ipfsd-ctl')
+
+const port = 9999
+const server = DaemonFactory.createServer(port)
+const df = DaemonFactory.create({ remote: true, port })
+server.start((err) => {
+  if (err) {
+    throw err
+  }
+
+  df.spawn((err, ipfsd) => {
+    if (err) {
+      throw err
+    }
+
+    ipfsd.api.id(function (err, id) {
+      console.log(id)
+      ipfsd.stop(server.stop)
+    })
+  })
+})
+```
+
+It's also possible to start the server from `.aegir` `pre` and `post` hooks. 
```

Comment:
Unnecessary line. It is going to confuse the users that have no idea about aegir to think they should use it.